src/state/apis/zapper/zapperApi.ts

```diff
@@ -0,0 +1,63 @@
+import { createApi } from '@reduxjs/toolkit/dist/query/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { ethChainId, toAssetId } from '@shapeshiftoss/caip'
+import { getConfig } from 'config'
+import qs from 'qs'
+import { BASE_RTK_CREATE_API_CONFIG } from 'state/apis/const'
+
+import { chainIdToZapperNetwork, createApiClient, zapperNetworkToChainId } from './client'
+
+const ZAPPER_BASE_URL = 'https://api.zapper.xyz'
+
+const authorization = `Basic ${Buffer.from(
+  `${getConfig().REACT_APP_ZAPPER_API_KEY}:`,
+  'binary',
+).toString('base64')}`
+
+const headers = {
+  accept: 'application/json',
+  authorization,
+}
+
+const zapperClient = createApiClient(ZAPPER_BASE_URL)
+
+type GetAppBalancesInput = {}
+type GetAppBalancesOutput = AssetId[]
+
+// https://docs.zapper.xyz/docs/apis/getting-started
+export const zapperApi = createApi({
+  ...BASE_RTK_CREATE_API_CONFIG,
+  reducerPath: 'zapperApi',
+  endpoints: build => ({
+    getZapperUniV2PoolAssetIds: build.query<GetAppBalancesOutput, GetAppBalancesInput>({
```

Comment:
```suggestion
    getZapperUniV2PoolAssetIds: build.query<GetAppBalancesOutput, void>({
```

smol requested change otherwise the consumption looks like (requires `{}` literal as an arg)
<img width="421" alt="image" src="https://user-images.githubusercontent.com/88504456/228387264-d3db53f1-00db-4ec4-8b7b-ad2dda4d314c.png">

Reply:
73a0224e8d1e0b685e23b6a55212b8fbb340dcb9